app/src/main/java/com/team7/cmput301/android/theirisproject/task/GetRecordPhotosTask.java

```diff
@@ -0,0 +1,56 @@
+package com.team7.cmput301.android.theirisproject.task;
+
+import android.os.AsyncTask;
+
+import com.team7.cmput301.android.theirisproject.IrisProjectApplication;
+import com.team7.cmput301.android.theirisproject.model.Record;
+import com.team7.cmput301.android.theirisproject.model.RecordPhoto;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import io.searchbox.core.Search;
+import io.searchbox.core.SearchResult;
+
+/**
+ * GetRecordPhotosTask queries the record photos associated to the problem
+ *
+ * @author jtfwong
+ * */
+public class GetRecordPhotosTask extends AsyncTask<String, List<RecordPhoto>, Void> {
+
+    private Callback cb;
+    private String queryString;
+
+    public GetRecordPhotosTask(Callback<List<RecordPhoto>> callback) {
+        this.cb = cb;
+    }
+
+    @Override
+    protected Void doInBackground(String... strings) {
+        new GetRecordListTask(new Callback<SearchResult>() {
+            @Override
+            public void onComplete(SearchResult res) {
+                List<RecordPhoto> recordPhotos = new ArrayList<>();
+                List<Record> recordList = res.getSourceAsObjectList(Record.class, true);
+                for (Record r: recordList) {
+                    Search search = new Search.Builder(queryString).addIndex(IrisProjectApplication.INDEX).addType("recordphoto").build();
```

Comment:
replace Search.Builder(queryString)... to Search.Builder(String.format(queryString, r.getId()))...